force-app/main/default/triggers/TaskTrigger.trigger

```diff
@@ -0,0 +1,20 @@
+trigger TaskTrigger on Task (before insert,before update,after insert,after update) {
+	
+    if(Trigger.isBefore){
+        if(Trigger.isInsert){
+        	System.debug('isBefore isInsert');
```

Comment:
You should make sure that the code you push doesnt include system.debugs nor console.logs